language-support/ts/codegen/tests/build-and-lint.sh

```diff
@@ -60,16 +60,10 @@ cd $TMP_DIR
 
 # Call daml2js.
 PATH=`dirname $YARN`:$PATH $DAML2TS -o daml2js $DAR
-# We only needed this package.json in order to provide resolution of
-# daml-types to daml2js and that's done with now.
-rm package.json
-# Since we eschew using workspaces and resolutions in this test, patch
-# up the daml-ledger dependency on daml-types.
-sed -i"" "s|\"@daml/types\": \"$SDK_VERSION\"|\"@daml/types\": \"file:../daml-types\"|" daml-ledger/package.json
```

Comment:
This really needs to become a proper Haskell test. I'll take care of that once this PR has landed.